video/out/opengl/egl_helpers.c

```diff
@@ -203,7 +203,8 @@ static bool create_context(struct ra_ctx *ctx, EGLDisplay display,
     }
 
     if (!egl_ctx) {
-        MP_MSG(ctx, msgl, "Could not create EGL context for %s!\n", name);
+        MP_MSG(ctx, msgl, "Could not create EGL context for %s (error=%u)!\n",
```

Comment:
`%d`? I think it is `EGLint`

Reply:
I guess this is wrong then:
```
video/out/hwdec/dmabuf_interop_gl.c
299:                    "Failed to import surface in EGL: %u\n", eglGetError());
```

Reply:
not a big deal, will work as long as size is the same, just print negative numbers differently... which we wouldn't get from this api anyway.